bhohnco/kingsCastle
# Added book info, book covers, and adjusted CSS for background color a…

…nd drop shadow

Is this a fix or a feature?
* 20+ Book Covers added with titles. 
What is the change or fix?
* Collections is close to complete, needs individual book info 
How should this be tested?
* Open the page and see how many books generate 
Linked Issues:
Add A Reviewer if Necessary
*
